maintenance/maintenance.html

```diff
@@ -1,155 +1,156 @@
+          <div class="mb-4">
+            <h2 class="h3-header">We Are Undergoing Maintenance</h1>
+          </div>
+          <p class="h5-black">Sorry, we're currently undergoing maintenance.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="donate-banner dark-theme py-5">
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h3 class="h3-heading mb-0">We all love the Web. Join Mozilla in defending it.</h3>
+            <h4 class="h3-heading mb-4">Let’s protect the world’s largest resource for future generations.</h4>
+            <div>
+              <a href="https://donate.mozilla.org/" target="_blank" rel="noopener noreferrer" c class="btn btn-secondary dark-theme">Donate now</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="site-footer dark-theme py-5">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-5 col-lg-6 d-flex flex-column justify-content-between">
+            <a class="logo" href="https://mozilla.org"><img src="static/images/mozilla-block-white.svg" width="96"></a>
+          </div>
+          <div class="col-md-6 col-lg-5 offset-md-1 d-flex flex-column justify-content-between">
+            <div class="row">
+              <div class="col-lg-6 col-xl-5">
+                <h5 class="h5-heading">More about us</h5>
+                <ul class="d-flex">
+                  <li class="mr-2"><a class="twitter-glyph small d-inline-block" href="https://twitter.com/mozilla"><span class="sr-only">Twitter</span></a></li>
+                  <li class="mr-2"><a class="instagram-glyph small d-inline-block" href="https://www.instagram.com/mozilla/"><span class="sr-only">Instagram</span></a></li>
+                  <li class="mr-2"><a class="github-glyph small d-inline-block" href="https://www.github.com/mozilla/foundation.mozilla.org/"><span class="sr-only">Github</span></a></li>
+                </ul>
+              </div>
+              <div class="col-lg-6 col-xl-7">
+                <ul class="link-list list-unstyled mb-0">
+                  <li class="mb-2"><a id="donate-footer-btn" href="https://donate.mozilla.org/" target="_blank" rel="noopener noreferrer" class="dark-theme">Donate</a></li>
+                  <li class="mb-2"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
+                  <li class="mb-2"><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="dark-theme">Legal</a></li>
+                  <li class="mb-2"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
+                  <li class="mb-2"><a href="https://foundation.mozilla.org/en/press-center/" class="dark-theme">Press Center</a></li>
+                  <li class="mb-2"><a href="https://mozilla.org/en-US/privacy/websites/" class="dark-theme">Privacy</a></li>
```

Comment:
This page is localized, and still accessible while we’re in maintenance mode 

```suggestion
                  <li class="mb-2"><a href="https://mozilla.org/privacy/websites/" class="dark-theme">Privacy</a></li>
```